eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -1,49 +1,85 @@
 parameters:
```

Comment:
FYI @benbp I know you are working on refactoring this file and so you will likely conflict with this PR.

Reply:
@weshaggard I chatted with Sean and was waiting for these changes to land first, hence why I did Java instead.